packages/back-end/src/controllers/datasources.ts

```diff
@@ -491,7 +491,8 @@ export async function postGoogleOauthRedirect(
   const context = getContextFromReq(req);
   const { projects } = req.body;
 
-  if (!context.permissions.canCreateDataSource({ projects })) {
+  // Huh, why? -- rb
+  if (!context.permissions.canCreateDataSource({ projects, type: undefined })) {
```

Comment:
This function is super old and not actually reachable in the UI anymore, but if we want to be correct here, the type should be `"google_analytics"`